src/Authoring/WinRT.Host/Directory.Build.targets

```diff
@@ -0,0 +1,23 @@
+<!--
+***********************************************************************************************
+Copyright (C) Microsoft Corporation. All rights reserved.
+***********************************************************************************************
+-->
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+    <Target Name="SplitWinRTHostIntoMUI"
+            Inputs="$(OutDir)WinRT.Host.dll"
+            Outputs="$(OutDir)WinRT.Host.dll.mui"
```

Comment:
Should WinRT.Host.dll also be an output here given we replace it or does just having the mui file as the output work out?

Reply:
Just having the mui seems to work just fine, but we can add it as an output